### PR TITLE
Add find order

### PR DESCRIPTION
**Pull request by D-Sinus**
(Put an `X` inside the `[ ]` to denote check mark `[X]`.)

- [X] **If creating a new file :**
  - [X] added links to it in the README files ?
  - [X] included tests with it ?
  - [X] added description (overview of algorithm, time and space complexity, and possible edge case) in docstrings ?

- [ ] **if done some changes :**
  - [ ] wrote short description in the PR explaining what the changes do ?
  - [ ] Fixes #[issue number] if related to any issue

- [ ] **other**

### CODE REVIEW

**Comment by D-Sinus** [comment]
I implemented algorithm that finds multiplicative order for given numbers.
If there's any problems in my code, please let me know.
Thanks.

**Comment by D-Sinus** [comment]
I implemented algorithm that finds multiplicative order for given numbers.
If there's any problems in my code, please let me know.
Thanks.

Ps. Cuz I'm not familiar with unittest, I've done some unnecessary commits.
     Sorry for that.

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 
 
-- [ ] **If creating a new file :**
-  - [ ] added links to it in the README files ?
-  - [ ] included tests with it ?
-  - [ ] added description (overview of algorithm, time and space complexity, and possible edge case) in docstrings ?
+- [X] **If creating a new file :**
+  - [X] added links to it in the README files ?
+  - [X] included tests with it ?
+  - [X] added description (overview of algorithm, time and space complexity, and possible edge case) in docstrings ?
```
Comment (keon) [inline PULL_REQUEST_TEMPLATE.md]: no need to edit template

Comment (D-Sinus) [reply PULL_REQUEST_TEMPLATE.md]: Hello.
First, thanks for your feedback.
I have question about 'edit template', did you mean that I have to erase that X mark
and leave PULL_REQUEST_TEMPLATE.md as same as original file?
I can reply in 3 mintues. Thanks.

Comment (D-Sinus) [reply PULL_REQUEST_TEMPLATE.md]: [Edit] Revert PULL_REQUEST_TEMPLATE.md

**Pull request merged**